docs/refs.bib

```diff
@@ -846,3 +846,25 @@ @misc{sykes2020boston
   title={- B 1000(Bk - 0.63)^2 where Bk is the proportion of blacks by town},
   url={https://github.com/scikit-learn/scikit-learn/issues/16155}
 }
+
+@misc{wikibooks2010monotone,
+	title = {Algorithm Implementation/Geometry/Convex hull/Monotone chain},
+  author={Wikibooks},
+	howpublished = {\url{https://en.wikibooks.org/wiki/Algorithm_Implementation/Geometry/Convex_hull/Monotone_chain}},
+	year = {2010},
+	note = {[Accessed 09-Jul-2023]},
+}
+
+@misc{barocas2017tutorial,
+  title={{NIPS} 2017 Tutorial on Fairness in Machine Learning},
+  author={Barocas, Solon and Hardt, Moritz},
+  url={https://fairmlbook.org/tutorial1.html},
+  year={2017}
+}
+
+@misc{mit2018gendershades,
```

Comment:
Should this be merged with `@inproceedings{buolamwini2018gender` ?

Reply:
Ah, I wondered about that. Since the reference I added is referring directly to the project website, how do you  suggest merging the two?

Reply:
Is this reference only cited in the plot_new_metrics file? If so, I think it'd make sense to remove this from refs.bib and cite the research paper in the prose instead. Let me know your thoughts.

Reply:
Can you add a URL field to the BibTeX entry?

Reply:
@alliesaizan yep only referenced cited in`plot_new_metrics.py` and for sure @riedgar-ms i'll update the `@inproceedings{buolamwini2018gender}` with a url field.